automated/linux/kselftest/kselftest.sh

```diff
@@ -206,6 +206,7 @@ elif [ -n "${TST_CMDFILES}" ]; then
 else
     ./run_kselftest.sh 2>&1 | tee "${LOGFILE}"
 fi
+parse_output
 # shellcheck disable=SC2164
 cd - || exit
-parse_output
```

Comment:
I guess you can keep parse_output here but replace `cd -` by `cd saved_pwd` where saved_pwd is the current working directory before the script potentially uses the `cd` command at the beginning of the script.

Reply:
Updated according suggestion, thanks!

Reply:
LGTM